corehq/apps/hqadmin/templates/hqadmin/user_audit_report.html

```diff
@@ -0,0 +1,14 @@
+{% extends "reports/bootstrap3/tabular.html" %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% block reportcontent %}
+  {% if warning_message %}
+    <div class="alert alert-warning">
+      <h4><i class="fa fa-exclamation-triangle"></i> {% trans "Warning" %}</h4>
```

Comment:
nit: we have a transition version of font-awesome on, but it is preferable to start using the new version of icons:
```
<i class="fa-solid fa-triangle-exclamation"></i>
```

Reply:
https://github.com/dimagi/commcare-hq/pull/36646